backend/data/src/main/kotlin/io/tolgee/component/machineTranslation/providers/BaiduTranslationProvider.kt

```diff
@@ -160,5 +273,72 @@ class BaiduTranslationProvider(
     // "rom"
     "ro" to "rom",
     // "hu"
+    "sq" to "alb",
+    "an" to "arg",
+    "ay" to "aym",
+    "os" to "oss",
+    "or" to "ori",
+    "ga" to "gle",
+    "ar-dz" to "arq",
+    "am" to "amh",
+    "az" to "aze",
+    "et" to "est",
+    "om" to "orm",
+    "oc" to "oci",
+    "ak" to "aka",
+    "as" to "asm",
+    "oj" to "oji",
+    "ba" to "bak",
+    "be" to "bel",
+    "fa" to "per",
+    "eu" to "baq",
+    "se" to "sme",
+    "is" to "ice",
+    "br" to "bre",
+    "pt-br" to "pot",
+    "bi" to "bis",
+    "bs" to "bos",
+    "cv" to "chv",
+    "ts" to "tso",
+    "da" to "dan",
+    "tt" to "tat",
+    "dv" to "div",
+    "sa" to "san",
+    "fo" to "fao",
+    "ff" to "ful",
+    "gu" to "guj",
+    "gn" to "grn",
+    "kg" to "kon",
+    "kl" to "kal",
+    "grc" to "gra",
+    "ka" to "geo",
+    "ha" to "hau",
+    "ky" to "kir",
+    "ca" to "cat",
+    "gl" to "glg",
+    "fr-ca" to "frn",
+    "co" to "cos",
+    "tlh" to "kli",
+    "ks" to "kas",
+    "kn" to "kan",
+    "kw" to "cor",
+    "cr" to "cre",
```

Comment:
I don't think this mapping is correct.
Looking at Baidu, they say that "cre" is code for language "克里克语". Looking up what "克里克语" means reveals the Muscogee language (Creek).
But you are mapping "cr" to that and "cr" is language code for the Cree language.
I guess it would be best to check with Baidu if their table is correct.

Reply:
I was mainly referring to https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes because attempting to search the language by their Chinese names would quickly subjects me to the rate limit of any search engines :(